### DISCUSSION TITLE
Detection of Adblockers by Youtube a big Change has been implemented,are you affected?

### DISCUSSION THREAD

**Discussion by Sabrehawk** [uBlockOrigin/uBlock-issues#2865] [Polls]
Im seeing widespread reports about the nuissance popup on all platforms and
yes have experienced myself in the exact timeframe as it became a public domain 
issue on social media (such as twitch etc).

I suspect a randomized detecion scheme to sabotage easy solutions for countermeasures.

The usual processes do not seem to resolve the random return of that pop up.

Youtube is making further agressive moves towards ramping up monetization.

**Comment by mapx-** [comment]
https://github.com/uBlockOrigin/uAssets/issues/19976
